app/views/landing/index.html.erb

```diff
@@ -5,6 +5,8 @@
   Having problems with the site? Email our system admin at unicorn_boss@rainbows.now</p>
 
 <%= link_to 'Sign Up', new_user_registration_path %>
+<%= link_to 'Log In', new_user_session_path %>
+<%= link_to "Sign out", destroy_user_session_path, :method => :delete %>
```

Comment:
please condition this
```
<% if current_user %>
  <%= link_to "Sign out", destroy_user_session_path, :method => :delete %>
<% else %>
  <%= link_to 'Sign Up', new_user_registration_path %>
  <%= link_to 'Log In', new_user_session_path %>
<% end %>
```
Can you see why this is good?

Reply:
Done!